src/web/routes/application/confirm.js

```diff
@@ -1,15 +1,26 @@
 const { handleRequestForPath } = require('./middleware')
 const { CONFIRM_URL } = require('./common/constants')
 
-const pageContent = ({ translate }) => ({
-  title: translate('confirm.title'),
-  subTitle: translate('confirm.subTitle')
+const confirmPageContent = req => ({
+  title: req.t('confirm.title'),
+  subTitle: req.t('confirm.subTitle')
+})
+
+const unsuccessfulApplicationPageContent = (req, eligibilityStatus) => ({
+  ...{
```

Comment:
I don't think the rest operator is needed here - just don't nest the properties.

Reply:
Done.